assets/styles/tools/_functions.scss

```
@@ -139,3 +139,24 @@
 }
 
 $context: 'frontend' !default;
+
+
+// Returns color code.
+//
+// ```scss
+// .c-box {
+//     width: color(primary);
+// }
+// ```
+//
+// @param  {string} $key - The color key in $colors
+// @return {function<string>} in hexadecimal
+@function color($key) {
+    @if map-has-key($colors, $key) {
+        @return map-get($colors, $key);
+    }
+
+    @warn "Unkown '#{$key}' in $colors.";
+
+    @return null;
+}
```

Fixed block comment tags, fail early, and fail hard instead of returning `null` which might break other areas and cause confusion.

```suggestion
// @param  {string} $key - The color key in $colors.
// @return {color}
@function color($key) {
    @if not map-has-key($colors, $key) {
        @error "Unknown '#{$key}' in $colors.";
    }

    @return map-get($colors, $key);
}
```

---

Done !